docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -176,6 +176,22 @@ For each Trino table which maps to BigQuery view there exists a system table whi
 Given a BigQuery view ``customer_view`` you can send query
 ``SELECT * customer_view$view_definition`` to see the SQL which defines view in BigQuery.
 
+.. _bigquery-sql-support:
+
+SQL support
+-----------
+
+The connector provides read and write access to data and metadata in
+the BigQuery database. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/create-table`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/drop-schema`
```

Comment:
@jhlodin Is there any sort rule here?

Reply:
Yes, the statements should reflect the grouping and ordering of SQL statements on this page - https://trino.io/docs/current/language/sql-support.html

So the ordering as they are in this PR right now is correct.